### PR TITLE
vai pagination tests - cluster events

### PR DESCRIPTION
**Pull request by yonasberhe23**
### Summary
Fixes #https://github.com/rancher/qa-tasks/issues/1380

Covers pagination functionality for cluster events list.

### Technical notes summary
To have deterministic tests for pagination, I used mock response (using cy.intercept) with lots of data hence the massive events data objects in `cypress/e2e/blueprints/explorer/cluster/events.js`. It's not pretty but, this is the easiest, most time efficient way to write these tests. Will revisit this when I have more time. https://github.com/rancher/qa-tasks/issues/1386

### Checklist
- [x] The PR is linked to an issue and the linked issue has a Milestone, or no issue is needed
- [x] The PR has a Milestone 
- [x] The PR template has been filled out
- [x] The PR has been self reviewed 
- [x] The PR has a reviewer assigned
- [x] The PR has automated tests or clear instructions for manual tests and the linked issue has appropriate QA labels, or tests are not needed
- [ ] The PR has reviewed with UX and tested in light and dark mode, or there are no UX changes

### CODE REVIEW

File: cypress/e2e/blueprints/explorer/cluster/events.js [hunk 1]
```diff
@@ -0,0 +1,3913 @@
+// GET /v1/events - small set of events data
+const eventsGetResponseSmallSet = {
+  type:         'collection',
+  links:        { self: 'https://yonasb29h.qa.rancher.space/v1/events' },
```
Comment (izaac) [inline cypress/e2e/blueprints/explorer/cluster/events.js:4]: Does the URL matter for this data to be visible in the UI?

Comment (yonasberhe23) [reply cypress/e2e/blueprints/explorer/cluster/events.js]: updated to a random prefix. still works the same

File: cypress/e2e/po/components/pagination.po.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  leftButton() {
+    return this.self().find('button:nth-child(2)');
+  }
+
+  rightButton() {
+    return this.self().find('button:nth-child(4)');
+  }
+
+  endButton() {
+    return this.self().find('button:nth-child(5)');
```
Comment (izaac) [inline cypress/e2e/po/components/pagination.po.ts:21]: Can't these be added as a `testid` property ?

**Review by izaac — COMMENTED** [review]
Just a nit about possibility to change the pag. buttons with test id locators. Looks good.

**Review by izaac — APPROVED** [review]

**Pull request merged**